recipes-asteroid/qml-asteroid/qml-asteroid_git.bb

```diff
@@ -11,34 +11,17 @@ PV = "+git${SRCPV}"
 S = "${WORKDIR}/git"
 inherit qmake5
 
-PACKAGE_ARCH = "${MACHINE_ARCH}"
-
 DEPENDS += "qtdeclarative qtsvg qtvirtualkeyboard mlite mapplauncherd-booster-qtcomponents qtdeclarative-native"
 RDEPENDS_${PN} += "qtsvg-plugins qtvirtualkeyboard asteroid-icons-ion"
 
-do_configure_prepend() {
-    if [ ${MACHINE_DISPLAY_FLAT_TIRE} ]
-    then
-        export EXTRA_QMAKEVARS_PRE="${EXTRA_QMAKEVARS_PRE} DEFINES+=FLAT_TIRE=${MACHINE_DISPLAY_FLAT_TIRE}"
-    fi
-    if [ ${MACHINE_DISPLAY_BORDER_GESTURE_WIDTH} ]
-    then
-        export EXTRA_QMAKEVARS_PRE="${EXTRA_QMAKEVARS_PRE} DEFINES+=BORDER_GESTURE_WIDTH=${MACHINE_DISPLAY_BORDER_GESTURE_WIDTH}"
-    fi
-    if [ ${MACHINE_DISPLAY_ROUND} = "true" ]
-    then
-        export EXTRA_QMAKEVARS_PRE="${EXTRA_QMAKEVARS_PRE} DEFINES+=ROUND_SCREEN"
-    fi
-    if [ ${MACHINE_HAS_WLAN} = "true" ]
-    then
-        export EXTRA_QMAKEVARS_PRE="${EXTRA_QMAKEVARS_PRE} DEFINES+=HAS_WLAN"
-    fi
-    if [ ${MACHINE_HAS_SPEAKER} = "true" ]
-    then
-        export EXTRA_QMAKEVARS_PRE="${EXTRA_QMAKEVARS_PRE} DEFINES+=HAS_SPEAKER"
+do_install_append() {
+    if [ -f ../machine.conf ]; then
+      install -d ${D}/etc/
+      cp ../machine.conf ${D}/etc/machine.conf
```

Comment:
As discussed in https://github.com/AsteroidOS/meta-sturgeon-hybris/pull/13/files#r500633781 I would prefer if we'd dynamically generate this file.